test-kernel.py

```diff
@@ -0,0 +1,23 @@
+# /// script
```

Comment:
I think this is another development file that can be removed? (I think I missed it the first time.)